README.md

```diff
@@ -161,7 +161,7 @@ the list of countries included in a macro-region (i.e., a continent or large reg
 
 ## The pyam package
 
-<img src="./_static/pyam-logo.png" width="133" height="100" align="right" alt="pyam
+<img src="./doc/source/_static/pyam-logo.png" width="133" height="100" align="right" alt="pyam
```

Comment:
In the latest pyam release, I included the logo directly in the pyam GitHub repo so that it could be retrieved directly (like here) and one wouldn't need to copy-paste it in every repo. See https://github.com/IAMconsortium/pyam/tree/main/doc/logos

Reply:
Ah sorry, I think I had an old version of pyam. Will change it!